discovery-provider/src/tasks/entity_manager/entity_manager.py

```diff
@@ -654,3 +676,31 @@ def get_entity_manager_events_tx(update_task, tx_receipt):
     return getattr(
         update_task.entity_manager_contract.events, MANAGE_ENTITY_EVENT_TYPE
     )().processReceipt(tx_receipt)
+
+
+def create_and_raise_indexing_error(err, redis, session):
+    logger.error(
+        f"Error in the indexing task at"
+        f" block={err.blocknumber} and hash={err.txhash}"
+    )
+    # set indexing error
+    set_indexing_error(redis, err.blocknumber, err.blockhash, err.txhash, err.message)
+
+    # seek consensus
+    has_consensus = confirm_indexing_transaction_error(
+        redis, err.blocknumber, err.blockhash, err.txhash, err.message
+    )
+    if not has_consensus:
+        # escalate error and halt indexing until there's consensus
```

Comment:
what if there's never consensus and we end up halting indexing forever bc one node is missing data and the other nodes don't have this problem?

Reply:
i think this change can only reduce the amount of halting because as it is now if a node can't process a tx for whatever reason, it'll halt forever. with this change, it can skip to a certain extent if it can reach consensus. there might be cases where a node by itself cannot progress and others can and in that case i think we should address the data discrepancies. i think that's a separate problem tho